README.md

```diff
@@ -1,6 +1,9 @@
 This is an automatically updated **experimental**
 version of Node.js [Node.js](https://github.com/nodejs/node)
```

Comment:
while you are here, can you please remove the duplicated "Node.js"?

Reply:
Of course!